src/connectors/geo-search/__tests__/connectGeoSearch-test.ts

```diff
@@ -660,7 +660,7 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/geo-search/
       }),
     ]);
 
-    // @ts-expect-error connector uses strings
+    // @ts-ignore
```

Comment:
Why finally ignoring?

Reply:
it errors with v3 but doesn't with v4, if we do expect-error it won't pass on v4